scripts/build.sh

```
@@ -0,0 +1,113 @@
+    info "Found VSCE version $VSCE_VERSION"
+fi
+
+# Step 3: Clean previous builds
+info "Cleaning previous builds..."
+if [ -d "themes" ]; then
+    rm -rf themes
+    info "Removed existing themes directory"
+fi
+
+if [ -f "*.vsix" ]; then
+    rm *.vsix
```

Shellcheck (0.10.0)

[info] Use ./*glob* or -- *glob* so names with dashes won't become options.

(SC2035)